pyfarm/agent/service.py

```diff
@@ -118,6 +115,60 @@ def agents_endpoint(cls):
         """
         return config["master_api"] + "/agents/"
 
+    @inlineCallbacks
+    def schedule_call(self, delay, function, *args, **kwargs):
+        """
+        Schedules ``function`` to be run after ``delay`` has elapsed.
```

Comment:
Maybe slightly misleading – by default, this will call ```function``` repeatedly and indefinitely every ```delay```. The behaviour described here can only be achieved by explicitly setting ```repeat``` to 1.